SubScript Actors basic implementation

anatoliykmetyuk opened the pull request.

Commit:
SubScript actor added. Still not tested due to conflicts with Akka

Commit:
SubScriptActor have passed elementary testing

Commit:
Actor's handler is deleted immediately after handling. VM now gets notified when messages are inserted

Commit:
Strange waiting loop removed from the debugger

Commit:
Plugin system added. Actors enhanced

AndreVanDelft commented:
I would like to have the SubScriptActor in package subscript.akka. 
Could you (Tolya) change the pull request accordingly?

Commit:
Activation doesn't stop after break is removed

Commit:
Activation resumes only after the correct child's start

Commit:
Activation resumes only after the correct child's start

Commit:
Loops now work correctly with optional breaks

Commit:
SubScript actor moved to subscript.akka package

anatoliykmetyuk commented:
I have added SubScriptActor to subscript.akka. Also, I have fixed the issue with breaks that were not working - now they seem to work fine in structures like:
foo . bar
foo / ..
foo / bar / . / ...
